Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -1251,6 +1253,7 @@ def update_record_command(client: Client, args: dict) -> Tuple[Any, Dict[Any, An
     record_id = str(args.get('id', ''))
     fields_str = str(args.get('fields', ''))
     custom_fields_str = str(args.get('custom_fields', ''))
+    input_display_value = args.get('fields', 'input_display_value') == 'true'
```

Comment:
Use `str2bool`

Reply:
str2bool is in the sdk, used `argToBoolean`